setup.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+
+"""
+Standard build script.
+"""
+
+from __future__ import print_function
+
+from os.path import dirname, join
+
+try:
+    from setuptools import setup, find_packages
+except ImportError:
+    print("You must have setuptools installed to use setup.py. Exiting...")
+    raise SystemExit(1)
+
+__docformat__ = 'restructuredtext'
+
+# Import requirements
+with open(join(dirname(__file__), 'requirements.txt')) as f:
+    required = f.read().splitlines()
+
+setup(
+    name="python-owasp-zap-v2.4",
```

Comment:
The Python library is using those names to prevent "incompatibility issues" between releases of ZAP. For example, newer versions include functionalities that will not work with older versions of ZAP, so the v2.4 (or v2.5) indicates the version of ZAP that the client implementation is targeting.